tests/src/test/scala/org/http4s/EntityEncoderSpec.scala

```diff
@@ -48,20 +48,21 @@ class EntityEncoderSpec extends Http4sSuite {
       "EntityEncoder should render streams with chunked transfer encoding without wiping out other encodings") {
       trait Foo
       implicit val FooEncoder: EntityEncoder[IO, Foo] =
-        EntityEncoder.encodeBy[IO, Foo](`Transfer-Encoding`(TransferCoding.gzip))(_ => Entity.empty)
+        EntityEncoder.encodeBy[IO, Foo](`Transfer-Encoding`(TransferCoding.gzip))(_ => Entity.Empty[IO]())
       implicitly[EntityEncoder[IO, Stream[IO, Foo]]].headers.get(`Transfer-Encoding`) match {
         case Some(coding: `Transfer-Encoding`) =>
           assertEquals(coding, `Transfer-Encoding`(TransferCoding.gzip, TransferCoding.chunked))
         case _ => fail("Match failed")
       }
     }
 
+/* Invalid With New Model What do we want to do about them?
```

Comment:
Isn't the equivalent to this one a chunked body with an empty stream?

Reply:
The thing is that this is checking the tranfer-encoding header for how it interacted with chunked. Which we no longer do in the EntityEncoder.